framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1030,15 +1039,15 @@ protected Boolean visitUnionSubtype(
     protected boolean visitWildcardSupertype(
             AnnotatedTypeMirror subtype, AnnotatedWildcardType supertype, VisitHistory visited) {
         if (supertype.isUninferredTypeArgument()) { //TODO: REMOVE WHEN WE FIX TYPE ARG INFERENCE
-            return isSubtype(subtype, supertype.getExtendsBound());
+            return supertype.atypeFactory.ignoreUninferredTypeArguments;
```

Comment:
Should this add `|| isSubtype(...)` to perform the previous subtype test if uninferred TAs should not be ignored?